src/handlers/int-test-support/helpers/s3Helper.ts

```diff
@@ -180,6 +180,21 @@ const getS3Objects = async (params: BucketAndPrefix): Promise<string[]> => {
   return content;
 };
 
+const deleteS3FolderBasedOnDate = async (
+  bucketName: string,
+  folderPrefix: string
+): Promise<boolean> => {
+  const result = await listS3Objects({ bucketName, prefix: folderPrefix });
+  const getDateFolderPrefix = result.Contents?.map((data) =>
+    data.Key?.slice(0, 27)
+  );
+  if (getDateFolderPrefix === undefined) return false;
+  for (const dateFolderPrefix of getDateFolderPrefix) {
```

Comment:
Isn't this logic just deleting all files beginning with `folderPrefix`? Or do the names need to be 27 characters long?
In any case it has a misleading name as I don't see any date-awareness in here. Just string prefix checking.

Reply:
Working on this with Nithya.